src/routes/routes.ts

```diff
@@ -78,13 +78,17 @@ export class Routes extends RouteCollection {
                             })
                         })
                     })
+
                     //Firewalls
                     // Old way restored. These routes has been disabled temporarily
                     /*router.prefix('/firewalls/:firewall(\\d+)', (router: RouterParser) => {
                         router.post('/compile', FirewallController, 'compile').name('firewalls.compile');
                         router.post('/install', FirewallController, 'install').name('firewalls.install');
                     });*/
 
+                    router.prefix('/colors', (router: RouterParser) => {
+                        router.get('/', FwCloudController, 'colors').name('fwclouds.colors');
+                    });
```

Comment:
If there is just one route, `prefix` is not required here. So you can replace it with:
```
router.get('/colors', FwCloudController, 'colors').name('fwclouds.colors');
```